official/vision/beta/modeling/backbones/mobilenet.py

```diff
@@ -0,0 +1,682 @@
+
+MNV3Large_BLOCK_SPECS = {
+    'spec_name': 'MobileNetV3Large',
+    'block_spec_schema': ['block_fn', 'kernel_size', 'strides', 'filters',
+                          'activation', 'se_ratio', 'expand_ratio',
+                          'use_normalization', 'use_bias'],
+    'block_specs': [
+        ('convbn', 3, 2, 16, 'hard_swish', None, None, True, False),
+
+        ('mbconv', 3, 1, 16, 'relu', None, 1., None, False),
+
+        ('mbconv', 3, 2, 24, 'relu', None, 4., None, False),
+        ('mbconv', 3, 1, 24, 'relu', None, 3., None, False),
+
+        ('mbconv', 5, 2, 40, 'relu', 1. / 4, 3., None, False),
+        ('mbconv', 5, 1, 40, 'relu', 1. / 4, 3., None, False),
+        ('mbconv', 5, 1, 40, 'relu', 1. / 4, 3., None, False),
+
+        ('mbconv', 3, 2, 80, 'hard_swish', None, 6., None, False),
+        ('mbconv', 3, 1, 80, 'hard_swish', None, 2.5, None, False),
+        ('mbconv', 3, 1, 80, 'hard_swish', None, 2.3, None, False),
+        ('mbconv', 3, 1, 80, 'hard_swish', None, 2.3, None, False),
+
+        ('mbconv', 3, 1, 112, 'hard_swish', 1. / 4, 6., None, False),
+        ('mbconv', 3, 1, 112, 'hard_swish', 1. / 4, 6., None, False),
+
+        ('mbconv', 5, 2, 160, 'hard_swish', 1. / 4, 6, None, False),
+        ('mbconv', 5, 1, 160, 'hard_swish', 1. / 4, 6, None, False),
+        ('mbconv', 5, 1, 160, 'hard_swish', 1. / 4, 6, None, False),
+
+        ('convbn', 1, 1, 960, 'hard_swish', None, None, True, False),
+        ('gpooling', None, None, None, None, None, None, None, None),
+        ('convbn', 1, 1, 1280, 'hard_swish', None, None, False, True),
+    ]
+}
+
+MNV3Small_BLOCK_SPECS = {
+    'spec_name': 'MobileNetV3Small',
+    'block_spec_schema': ['block_fn', 'kernel_size', 'strides', 'filters',
+                          'activation', 'se_ratio', 'expand_ratio',
+                          'use_normalization', 'use_bias'],
+    'block_specs': [
+        ('convbn', 3, 2, 16, 'hard_swish', None, None, True, False),
+
+        ('mbconv', 3, 2, 16, 'relu', 1. / 4, 1, None, False),
+
+        ('mbconv', 3, 2, 24, 'relu', None, 72. / 16, None, False),
+        ('mbconv', 3, 1, 24, 'relu', None, 88. / 24, None, False),
+
+        ('mbconv', 5, 2, 40, 'hard_swish', 1. / 4, 4., None, False),
+        ('mbconv', 5, 1, 40, 'hard_swish', 1. / 4, 6., None, False),
+        ('mbconv', 5, 1, 40, 'hard_swish', 1. / 4, 6., None, False),
+
+        ('mbconv', 5, 1, 48, 'hard_swish', 1. / 4, 3., None, False),
+        ('mbconv', 5, 1, 48, 'hard_swish', 1. / 4, 3., None, False),
+
+        ('mbconv', 5, 2, 96, 'hard_swish', 1. / 4, 6., None, False),
+        ('mbconv', 5, 1, 96, 'hard_swish', 1. / 4, 6., None, False),
+        ('mbconv', 5, 1, 96, 'hard_swish', 1. / 4, 6., None, False),
+
+        ('convbn', 1, 1, 576, 'hard_swish', None, None, True, False),
+        ('gpooling', None, None, None, None, None, None, None, None),
+        ('convbn', 1, 1, 1024, 'hard_swish', None, None, False, True),
+    ]
+}
+
+MNV3EdgeTPU_BLOCK_SPECS = {
```

Comment:
Any paper/doc to explain that architecture?

Reply:
There is no paper/doc out there. But i have added the link to the original slim version.

Reply:
sg, thanks.